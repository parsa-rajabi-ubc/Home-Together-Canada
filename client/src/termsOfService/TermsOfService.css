.tos-page-layout {
    @apply p-2 p-8 m-20 bg-white rounded-xl shadow-xl;
}

.page-title {
    @apply mb-4 text-2xl font-bold;
}

.point-title {
    @apply text-lg font-semibold;
}

.sub-point {
    @apply ml-16;
}

.text {
    @apply font-light;
}