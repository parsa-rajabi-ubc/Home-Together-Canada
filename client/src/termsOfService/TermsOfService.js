/**
 * @Author:     Rachelle Gelden / Parsa Rajabi
 * @Created:    2021.01.13
 *
 * @Description: Terms of service page
 *
 */

import React from 'react';
import {TERMS_OF_SERVICE} from "../common/constants/termsOfServiceText";

function TermsOfService() {
    return (
        <div className={"tos-page-layout"}>
            <h1 className={"page-title"}>{TERMS_OF_SERVICE.title}</h1>
            <p className={"text"}>{TERMS_OF_SERVICE.subParagraph1}</p>
            <br/>
            <p className={"text"}>{TERMS_OF_SERVICE.subParagraph2}</p>
            <br/>

            <h2 className={"point-title"}>{TERMS_OF_SERVICE.point1Title}</h2>
            <p className={"text"}>{TERMS_OF_SERVICE.point1Text}</p>
            <br/>

            <h2 className={"point-title"}>{TERMS_OF_SERVICE.point2Title}</h2>
            <p className={"text"}>{TERMS_OF_SERVICE.point2Text}</p>
            <br/>

            <h2 className={"point-title"}>{TERMS_OF_SERVICE.point3Title}</h2>
            <p className={"text"}>{TERMS_OF_SERVICE.point3Text1}</p>
            <br/>

            <p className={"sub-point text"}>{TERMS_OF_SERVICE.point3SubpointA}</p>
            <p className={"sub-point text"}>{TERMS_OF_SERVICE.point3SubpointB}</p>
            <p className={"sub-point text"}>{TERMS_OF_SERVICE.point3SubpointC}</p>
            <br/>

            <p className={"text"}> {TERMS_OF_SERVICE.point3Text2}</p>
            <br/>

            <h2 className={"point-title"}>{TERMS_OF_SERVICE.point4Title}</h2>
            <p className={"text"}>{TERMS_OF_SERVICE.point4Text}</p>
            <br/>

            <h2 className={"point-title"}>{TERMS_OF_SERVICE.point5Title}</h2>
            <p className={"text"}>{TERMS_OF_SERVICE.point5Text}</p>
            <br/>

            <h2 className={"point-title"}>{TERMS_OF_SERVICE.point6Title}</h2>
            <p className={"text"}>{TERMS_OF_SERVICE.point6Text}</p>
            <br/>

            <h2 className={"point-title"}>{TERMS_OF_SERVICE.point7Title}</h2>
            <p className={"text"}>{TERMS_OF_SERVICE.point7Text}</p>
            <br/>

            <h2 className={"point-title"}>{TERMS_OF_SERVICE.point8Title}</h2>
            <p className={"text"}>{TERMS_OF_SERVICE.point8Text}</p>
            <br/>

            <h2 className={"point-title"}>{TERMS_OF_SERVICE.point9Title}</h2>
            <p className={"text"}>{TERMS_OF_SERVICE.point9Text}</p>
            <br/>

            <p className={"sub-point text"}>{TERMS_OF_SERVICE.point9SubpointA}</p>
            <p className={"sub-point text"}>{TERMS_OF_SERVICE.point9SubpointB}</p>
            <p className={"sub-point text"}>{TERMS_OF_SERVICE.point9SubpointC}</p>
            <p className={"sub-point text"}>{TERMS_OF_SERVICE.point9SubpointD}</p>
            <p className={"sub-point text"}>{TERMS_OF_SERVICE.point9SubpointE}</p>
            <p className={"sub-point text"}>{TERMS_OF_SERVICE.point9SubpointF}</p>
            <p className={"sub-point text"}>{TERMS_OF_SERVICE.point9SubpointG}</p>
            <p className={"sub-point text"}>{TERMS_OF_SERVICE.point9SubpointH}</p>
            <br/>


            <h2 className={"point-title"}>{TERMS_OF_SERVICE.point10Title}</h2>
            <p className={"text"}>{TERMS_OF_SERVICE.point10Text}</p>
            <br/>

            <p className={"sub-point text"}>{TERMS_OF_SERVICE.point10SubpointA}</p>
            <p className={"sub-point text"}>{TERMS_OF_SERVICE.point10SubpointB}</p>
            <p className={"sub-point text"}>{TERMS_OF_SERVICE.point10SubpointC}</p>
            <br/>

            <h2 className={"point-title"}>{TERMS_OF_SERVICE.point11Title}</h2>
            <p className={"text"}>{TERMS_OF_SERVICE.point11Text}</p>
            <br/>

            <p className={"sub-point text"}>{TERMS_OF_SERVICE.point11SubpointA}</p>
            <p className={"sub-point text"}>{TERMS_OF_SERVICE.point11SubpointB}</p>
            <p className={"sub-point text"}>{TERMS_OF_SERVICE.point11SubpointC}</p>
            <p className={"sub-point text"}>{TERMS_OF_SERVICE.point11SubpointD}</p>
            <br/>

            <h2 className={"point-title"}>{TERMS_OF_SERVICE.point12Title}</h2>
            <p className={"text"}>{TERMS_OF_SERVICE.point12Text}</p>
            <br/>

            <h2 className={"point-title"}>{TERMS_OF_SERVICE.point13Title}</h2>
            <p className={"text"}>{TERMS_OF_SERVICE.point13Text}</p>
            <br/>

            <h2 className={"point-title"}>{TERMS_OF_SERVICE.point14Title}</h2>
            <p className={"text"}>{TERMS_OF_SERVICE.point14Text}</p>
            <br/>

            <h2 className={"point-title"}>{TERMS_OF_SERVICE.point15Title}</h2>
            <p className={"text"}>{TERMS_OF_SERVICE.point15Text}</p>
            <br/>

            <h2 className={"point-title"}>{TERMS_OF_SERVICE.point16Title}</h2>
            <p className={"text"}>{TERMS_OF_SERVICE.point16Text}</p>
            <br/>

            <h2 className={"point-title"}>{TERMS_OF_SERVICE.point17Title}</h2>
            <p className={"text"}>{TERMS_OF_SERVICE.point17Text}</p>
            <br/>

            <h2 className={"point-title"}>{TERMS_OF_SERVICE.point18Title}</h2>
            <p className={"text"}>{TERMS_OF_SERVICE.point18Text}</p>
            <br/>

            <h2 className={"point-title"}>{TERMS_OF_SERVICE.point19Title}</h2>
            <p className={"text"}>{TERMS_OF_SERVICE.point19Text}</p>
            <br/>

            <h2 className={"point-title"}>{TERMS_OF_SERVICE.point20Title}</h2>
            <p className={"text"}>{TERMS_OF_SERVICE.point20Text}</p>
            <br/>

            <h2 className={"point-title"}>{TERMS_OF_SERVICE.point21Title}</h2>
            <p className={"text"}>{TERMS_OF_SERVICE.point21Text}</p>
            <br/>

            <h2 className={"point-title"}>{TERMS_OF_SERVICE.point22Title}</h2>
            <p className={"text"}>{TERMS_OF_SERVICE.point22Text}</p>
            <br/>

            <h2 className={"point-title"}>{TERMS_OF_SERVICE.point23Title}</h2>
            <p className={"text"}>{TERMS_OF_SERVICE.point23Text}</p>
            <br/>

            <h2 className={"point-title"}>{TERMS_OF_SERVICE.point24Title}</h2>
            <p className={"text"}>{TERMS_OF_SERVICE.point24Text}</p>
            <br/>

            <h2 className={"point-title"}>{TERMS_OF_SERVICE.point25Title}</h2>
            <p className={"text"}>{TERMS_OF_SERVICE.point25Text}</p>
            <br/>

            <h2 className={"point-title"}>{TERMS_OF_SERVICE.point26Title}</h2>
            <p className={"text"}>{TERMS_OF_SERVICE.point26Text}</p>
            <br/>

            <h2 className={"point-title"}>{TERMS_OF_SERVICE.point27Title}</h2>
            <p className={"text"}>{TERMS_OF_SERVICE.point27Text}</p>
            <br/>

            <h2 className={"point-title"}>{TERMS_OF_SERVICE.point28Title}</h2>
            <p className={"text"}>{TERMS_OF_SERVICE.point28Text}</p>
            <br/>

            <h2 className={"point-title"}>{TERMS_OF_SERVICE.point29Title}</h2>
            <p className={"text"}>{TERMS_OF_SERVICE.point29Text}</p>
            <br/>
        </div>
    );
}

export default TermsOfService;