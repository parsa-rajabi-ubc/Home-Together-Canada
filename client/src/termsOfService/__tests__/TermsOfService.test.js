/**
 * @Author:     Parsa Rajabi
 * @Created:    2021.1.19
 *
 * @Description: Terms of Service Component snapshot test. Returns a snapshot.
 *
 */

import React from "react";
import TermsOfService from "../TermsOfService";
import renderer from "react-test-renderer";

describe('TermsOfService', () => {

    describe('Snapshot Test', () => {
        it("should render correctly regardless of properties", () => {
            // given

            //when
            const component = renderer.create(<TermsOfService/>);
            const tree = component.toJSON();
            //then
            expect(tree).toMatchSnapshot();
        });
    });
});