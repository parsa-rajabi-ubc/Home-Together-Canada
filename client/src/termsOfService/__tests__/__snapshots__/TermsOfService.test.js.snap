// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TermsOfService Snapshot Test should render correctly regardless of properties 1`] = `
<div
  className="tos-page-layout"
>
  <h1
    className="page-title"
  >
    Terms of Service

  </h1>
  <p
    className="text"
  >
    Home Together Canada provides individuals who wish to enter into a shared living experience, the basic tools needed to find others that may wish to share a living experience with them and we list information with links to organizations, groups and businesses that provide housing and services that support shared living. 

  </p>
  <br />
  <p
    className="text"
  >
    The tools and information we provide are available at hometogether.ca and can be accessed on mobile, tablet, and other smart devices.
The Website is owned by Home Together Canada Assoc. which is a volunteer run non profit.  These Terms of Service together with our Privacy Policy, govern your relationship with us. If you use or access the Website, or Services, you are indicating that you have read, understand, and agree to comply with and be legally bound by these Terms, whether or not you become a registered user of the Site. If you do not agree to these Terms, then you do not have the right to access or use the Website, or Services.
If you agree to these Terms on behalf of a society or other legal entity, you represent and warrant that you have the authority to bind that company or other legal entity to these Terms and, in such event, “you” and “your” will refer and apply to that company or other legal entity.

  </p>
  <br />
  <h2
    className="point-title"
  >
    1. Terms of Service Subject to Amendment

  </h2>
  <p
    className="text"
  >
    These Terms are subject to change without notice. By continuing to use or access the Website and Services, you agree to be bound by any such changes. Unless explicitly stated otherwise, any new features or functionality that augment or enhance the Website or Services will be subject to these Terms.

  </p>
  <br />
  <h2
    className="point-title"
  >
    2. The Services

  </h2>
  <p
    className="text"
  >
    Home Together Canada Assoc. has compiled and posted sharing ideas and sharing options on the website for members to read and utilize if they choose. The website provides members the ability to create a short basic profile so that they can search for and be searched for by others with profiles on the site.  The website also provides information and links to sharing services and businesses that provide services to those wishing to enter into a shared living experience. We provide this service to you so that you can find sharing services and housing opportunities you may be looking for.  Individuals can anonymously message each other through the site. All individuals using the site are responsible for deciding when they give another person their real name,  how they go forward , when and where they meet, and what steps they choose to take to enter into Living Arrangements with others. They do this directly with each other. We do not own, operate, manage, provide, rent, or control any properties, including the Bedrooms or Residences listed by members on the hometogether.ca  website or offered in any of the subcategories listed in the Classifieds
By using the Website and Services, you understand and acknowledge that we provide you the basic tool to find others and information about sharing business and service groups so you are able to find them. Our responsibilities are limited to facilitating the availability of the Website only.
We are not a party to any agreements entered into between any individual, business, group, organization or other entity of any kind. We have no control over the conduct of members or the service providers listed on the Website,  and disclaim all liability in this regard to the maximum extent permitted by law.

  </p>
  <br />
  <h2
    className="point-title"
  >
    3. Account Registration

  </h2>
  <p
    className="text"
  >
    In order to access certain features on the site you must register to the site .
If you choose to register to become a member or register to be an advertiser, you represent that you are of legal age to form a binding contract and are not a person barred from receiving services under the laws of Canada or other applicable jurisdictions. You also agree to:

  </p>
  <br />
  <p
    className="sub-point text"
  >
    a) provide true, accurate, current, and complete information about yourself as required by our registration form 

  </p>
  <p
    className="sub-point text"
  >
    b) maintain and update the Registration Data to keep it true, accurate, current, and complete; and

  </p>
  <p
    className="sub-point text"
  >
    c) be responsible for all activities that occur under your Account registration.

  </p>
  <br />
  <p
    className="text"
  >
     
    If you provide any information that is, or we have reason to suspect is, untrue, inaccurate, not current, incomplete, or otherwise in violation of these Terms, we have the right to suspend or terminate your Account and refuse any and all current or future use of the Services.
You are responsible for safeguarding your password. You agree that you will not disclose your password to any third party.  You are not permitted to share your Account with anyone or allow others to access or use your Account. You must immediately notify us of any unauthorized use of your Account.
To search for others on the site, you must be registered as a member, you will need to provide certain information about yourself including your name and contact information as well as certain information about your preferences in relation to a Living Arrangement. 

  </p>
  <br />
  <h2
    className="point-title"
  >
    4. Residential Tenancy Act and Regulations

  </h2>
  <p
    className="text"
  >
    You acknowledge that the nature of the Living Arrangement constitutes a living accommodation in which a home sharer shares a bathroom or kitchen facilities with a home owner, and as such, pursuant to the Residential Tenancy Act (British Columbia), currently the residential tenancy laws and regulations in some Provinces do not apply to shared living Arrangements. NOTE that Regulations in all Provinces change over time, and it is your responsibility to check with your province to ensure you have up to date information about their tenancy regulations before entering into a shared living agreement with others.

  </p>
  <br />
  <h2
    className="point-title"
  >
    5. 
  </h2>
  <p
    className="text"
  >
    Home Together Canada charges a one-time membership fee that remains in place for as long as you remain a member on the website.  If you choose to have your registration information and profile deleted from the site you forfeit your membership and will have to pay another membership fee if you choose to register to the site at some time in the future.

  </p>
  <br />
  <h2
    className="point-title"
  >
    6. 
  </h2>
  <p
    className="text"
  >
    Home Together Canada is an information only site and takes no part in the shared living agreements individuals create between themselves or other entities.  Home Together Canada has no control over the conduct of members or the service providers listed on the Website or the agreements they form between them. Home Together Canada takes no responsibility for the outcome of these agreements and disclaim all liability in this regard to the maximum extent permitted by law.

  </p>
  <br />
  <h2
    className="point-title"
  >
    7. Feedback and Ratings

  </h2>
  <p
    className="text"
  >
    Home together Canada provides those registered to the site to post public comments and star ratings about businesses, services and each other. Feedback and Ratings reflect the opinions of individual Members and do not reflect the opinion of Home Together Canada. We make no representations about the accuracy of such Feedback or Ratings. Ratings are part of a Member’s public profile such as on a member home share listings in the classified section, together with other relevant information.

  </p>
  <br />
  <h2
    className="point-title"
  >
    8. Ownership of Content

  </h2>
  <p
    className="text"
  >
    All material on the Website and any related registered domains and sub-domains, and any other digital medium owned or operated by us, including the design, layout, videos, audio, text, photographs, graphics, interfaces, computer code, appearance, trademarks, and logos used therein as well as individual articles, blogs, videos, photographs, and other content or elements comprising the Website are owned by us, unless otherwise specified.
Reproduction of any part thereof, is prohibited except with our written consent or in accordance with the law 

  </p>
  <br />
  <h2
    className="point-title"
  >
    9. Your Representations and Warranties

  </h2>
  <p
    className="text"
  >
    By submitting, posting, uploading, or transmitting content on or through the Website, Application, or Services in any format including comments, images, photographs, graphics, texts, videos, audio, appearance, trademarks, or logos used therein, you represent and warrant that you own or have the necessary licences, rights, consents, and permissions to use and authorize us to use such Member Content in the manner contemplated by us and these Terms.
You agree not to submit or use any inappropriate content in relation to the Website, Application, or Services. In furtherance of the above, you agree that you will not:

  </p>
  <br />
  <p
    className="sub-point text"
  >
    submit, post, upload, or transmit material that is copyrighted, protected by trademark, or otherwise subject to third party proprietary rights, including privacy and publicity rights, unless you are the owner of such rights or have permission from their rightful owner to post the material and to grant us all of the rights granted herein;

  </p>
  <p
    className="sub-point text"
  >
    submit, post, or upload falsehoods or misrepresentations that could damage us, our users, or any third party;

  </p>
  <p
    className="sub-point text"
  >
    impersonate any person or entity or falsely state or otherwise misrepresent your affiliation with a person or entity;

  </p>
  <p
    className="sub-point text"
  >
    submit, post, or upload material that is unlawful, obscene, defamatory, libelous, threatening, pornographic, harassing, hateful, racially or ethnically offensive, or encourages conduct that would be considered a criminal offence, give rise to a civil liability, violate any local, provincial, national, or international law, or is otherwise inappropriate;

  </p>
  <p
    className="sub-point text"
  >
    submit, post, or upload advertisements or solicitations of business, unless in the classifieds section

  </p>
  <p
    className="sub-point text"
  >
    collect or store personal data about Members or other users without their express consent;

  </p>
  <p
    className="sub-point text"
  >
    manipulate the Feedback or Ratings system in any manner, such as instructing a third party to submit a particular Rating about another Member; or

  </p>
  <p
    className="sub-point text"
  >
    submit, post, or upload material that contains anything, such as viruses, that will disturb, destroy, or impair the use of the Website, Application, or Services.

  </p>
  <br />
  <h2
    className="point-title"
  >
    10. Responsibility for Content, Actions, and Compliance with Laws
  </h2>
  <p
    className="text"
  >
    You acknowledge and agree that you are responsible for your own acts and omissions and are also responsible for the acts and omissions of any individuals who represent you, whether as an employee, contractor, agent, or in another capacity, and any individuals who reside at or are otherwise present at your Residence at your request or invitation.You understand and agree that we do not act as an employer, insurer, or agent of any Member. We do not have any control over the conduct of any Member and will not be liable, in any way for any acts or omissions of any Member. We do not make any guarantees with respect to the suitability of the Living Arrangement made between members or offered by any group, business, organization or other entity listed on the site.You agree that you will have no recourse against us for any alleged or actual infringement or misappropriation of any proprietary or other right in Member Content submitted by you. You assume all liability for any claims, suits, or grievances filed against you, including all damages related to your submitted Member Content and your participation on and through the Website, Application, and Services.You are responsible for complying with all applicable laws and regulations, including human rights laws, criminal laws, intellectual property laws, housing laws and regulations, health and safety laws, regulations, and standards, building codes and standards, and bylaws and other municipal regulations when you use the Website, Application, and Services. You will be solely responsible for the breach of any laws in association with your use of the Website, Application, and Services.Home owners acknowledge and agree;
  </p>
  <br />
  <p
    className="sub-point text"
  >
    not to breach any agreements they have entered into with any third parties including homeowners’ associations, condominium or strata boards, or insurance providers; and

  </p>
  <p
    className="sub-point text"
  >
    to comply with all applicable laws including zoning and other applicable bylaws (including having all required permits, licenses, and registrations).

  </p>
  <p
    className="sub-point text"
  >
    Any agreement entered into is strictly between the owner and home sharer(s)  and we are not a party to it.

  </p>
  <br />
  <h2
    className="point-title"
  >
    11. Prohibited Activities
  </h2>
  <p
    className="text"
  >
    Members are prohibited from requesting, offering, negotiating, or engaging in any activities that:
  </p>
  <br />
  <p
    className="sub-point text"
  >
    violate applicable laws or regulations or the legal rights of any person or entity including human rights laws, criminal laws, intellectual property laws, housing laws and regulations, health and safety laws, regulations, and standards, building codes and standards, and bylaws and other municipal regulations;
  </p>
  <p
    className="sub-point text"
  >
    encourage or facilitate illegal activity;
  </p>
  <p
    className="sub-point text"
  >
    would harm any person including children or minors; or
  </p>
  <p
    className="sub-point text"
  >
    relate to spam, or unsolicited services or products, or false, misleading, or fraudulent services or products.
  </p>
  <br />
  <h2
    className="point-title"
  >
    12. No Endorsement
  </h2>
  <p
    className="text"
  >
    We do not endorse any members, businesses, organization, services or other entities on the website, nor will we be responsible for their actions or inactions. You should therefore ensure that you carry out your own searches and ask questions of anyone that you wish to enter into a Living Arrangement with in order to satisfy yourself as to the suitability of the Living Arrangement you are creating and your compatibility, as applicable.

  </p>
  <br />
  <h2
    className="point-title"
  >
    13. Opinions Expressed on Website and Application
  </h2>
  <p
    className="text"
  >
    Opinions and other statements expressed by Members and third parties (including bloggers and feature contributors) are theirs alone, and do not represent our opinions. Content created by third parties is the sole responsibility of the third parties and its accuracy and completeness are not endorsed or guaranteed by us. You acknowledge that by providing you with the ability to view and distribute Member Content through the Website, Application, and Services, we are not undertaking any obligation or liability relating to the Member Content. We do not undertake or assume any duty to monitor the Website and Application for inappropriate or unlawful content and we assume no responsibility or liability which may arise from the content thereof, including claims for defamation, libel, slander, infringement, invasion of privacy and publicity rights, obscenity, pornography, profanity, fraud, or misrepresentation. Notwithstanding the foregoing, we reserve the right to block or remove communications, postings, or materials at any time in our sole discretion.

  </p>
  <br />
  <h2
    className="point-title"
  >
    14. Disclaimer
  </h2>
  <p
    className="text"
  >
    The information contained in the Website and Application is for general information purposes only. While we use reasonable efforts to include accurate information on the Website, we make no representations or warranties of any kind, express or implied, that any content is or remains complete, accurate, up-to-date, available, free from bugs, errors or omissions, or fit or suitable for any purpose. Any reliance you place on such information is strictly at your own risk. None of the information on the Website or Application is to be taken as advice of any kind. The Services are made available to you on an “as is” basis. We expressly deny any and all warranties and representations, express or implied, regarding the Services. For greater certainty, we do not represent nor warrant that the use of the Website or Services will satisfy your purposes. It will be your own responsibility to ensure that any services available through the Website and Application meet your specific requirements. While we make reasonable efforts to ensure that the Website and services are available, we do not represent, warrant, or guarantee in any way the Website's continued availability at all times or uninterrupted use by you of the Website or Services.

  </p>
  <br />
  <h2
    className="point-title"
  >
    15. Limitation of Liability
  </h2>
  <p
    className="text"
  >
    We take no responsibility for, and will not be liable for, any error or inaccuracies in any content or the Website or Services being temporarily unavailable due to technical issues beyond our control. It is our right to make the Website or Services unavailable from time to time, solely at our discretion, and we will not be liable for any loss or damage relating to such unavailability. We are not responsible for any loss or damage, including loss or damage arising from the loss of data, revenue, salary, profits, or reputation in connection with the use of the Website or Services. Any claim against us will be limited to the amount you paid, if any, for use of the Website, and Services.

  </p>
  <br />
  <h2
    className="point-title"
  >
    16. Third Party Websites
  </h2>
  <p
    className="text"
  >
    We provide information and links to other websites. These links are provided for your convenience to provide further information. They do not signify that we endorse the website(s). We have no responsibility for the content of the linked website(s). You agree that we will not be liable in any way, either directly or indirectly, for any loss or damage of any kind incurred as a result of, or in connection with the use of or reliance on any content, goods or services available through such third parties.

  </p>
  <br />
  <h2
    className="point-title"
  >
    17. International Use
  </h2>
  <p
    className="text"
  >
    This website is for the use of Canadians wishing to enter into a shared living arrangement with others in Canada and for sharing businesses, groups and service providers in Canada. We make no representation that materials on this site are appropriate or available for use in locations outside Canada. If you access the Website, or Services from other locations, you agree that you are doing so on your own initiative and that you are wholly responsible for compliance with local laws.
  </p>
  <br />
  <h2
    className="point-title"
  >
    18. Changes to the Services
  </h2>
  <p
    className="text"
  >
    In connection with any purchase of Services from the Website or Application, we do not make any promise regarding the continuation of any current features or functionality or delivery of any future functionality or features. We have the right to change any of the features on, or the services being offered through, the Website and Application.

  </p>
  <br />
  <h2
    className="point-title"
  >
    19. Discontinuation or Suspension of Website, Application, or Services
  </h2>
  <p
    className="text"
  >
    We reserve the right at any time and from time to time to modify, suspend, or discontinue, temporarily or permanently, the Website, Application, or Services, or any part thereof, with or without notice and in our sole discretion. You agree that we will not be liable to you or any third party for any such modification, suspension, or discontinuance of the Website, Application, or Services.
  </p>
  <br />
  <h2
    className="point-title"
  >
    20. Termination, Unauthorized Use
  </h2>
  <p
    className="text"
  >
    In the event that you breach these Terms, we have the right to terminate your access to the Website, Application, and Services, to delete any Member Content, and to disclose any of your information as necessary. Unauthorized use of the Website, Application, or Services may give rise to a claim for damages and/or result in you being held liable for a criminal offence in a court of law. In addition, we reserve the right to terminate your access to our Website, Application, and Services for any reason, and to take any other actions that we, in our sole discretion, believe to be in the interest of us or our users as a whole.
  </p>
  <br />
  <h2
    className="point-title"
  >
    21. Indemnity
  </h2>
  <p
    className="text"
  >
    You agree to indemnify and hold harmless Home Together Canada Assoc.  and their respective directors, officers,  employees, and licensors, from and against all losses, expenses, damages, and costs, including reasonable lawyers’ fees, resulting from your acts, omissions, access to or use of the Website, Application, or Services, your violation of these Terms, your breach of your representations and warranties hereunder, or your violation, or violation by any other user under your name, of any applicable laws or regulations, or the legal rights of any person or entity in relation to your use of the Website, Application, or Services. We reserve the right to take over the exclusive defense of any claim for which we are entitled to indemnification, in our sole discretion. In such event, you must provide us with such cooperation as is reasonably requested by us.
  </p>
  <br />
  <h2
    className="point-title"
  >
    22. Relationship
  </h2>
  <p
    className="text"
  >
    These Terms do not create a joint venture, partnership, employment, or agency relationship between us and you. If you choose to create an Account and register to become a member or advertiser on the hometogether.ca  website, you understand and agree that your relationship with us is limited to being a Member and not our employee, agent, joint venture, or partner. You act exclusively on your own behalf and for your own benefit, and not on our behalf or for our benefit. We do not control, and have no right to control, your Account, your offline activities associated with you creating your own shared living arrangement and agreements. As a Member, you agree not to do anything to create a false impression that you are endorsed by, employed by, partnering with, or acting on behalf of or for the benefit of us, including by inappropriately using any of our intellectual property including terms, trademark, or branding.
  </p>
  <br />
  <h2
    className="point-title"
  >
    23. Privacy
  </h2>
  <p
    className="text"
  >
    Registration data and certain other information that we collect about you are subject to our Privacy Policy. See Privacy Policy
  </p>
  <br />
  <h2
    className="point-title"
  >
    24. Headings
  </h2>
  <p
    className="text"
  >
    The headings used in these Terms are inserted for convenience of reference only and will not affect the construction or meaning of any of the provisions contained in these Terms or govern the rights or liabilities of any of the parties subject to these Terms.
  </p>
  <br />
  <h2
    className="point-title"
  >
    25. Severability
  </h2>
  <p
    className="text"
  >
    The invalidity or unenforceability of any provision of these Terms will not affect the validity or enforceability of any other provision of these Terms and any such invalid or unenforceable provision will be deemed to be severable.

  </p>
  <br />
  <h2
    className="point-title"
  >
    26. Waiver
  </h2>
  <p
    className="text"
  >
    No waiver of any breach of any provision of these Terms will constitute a waiver of any prior, concurrent, or subsequent breach of the same or any other provisions hereof, and no waiver will be effective unless made in writing and signed by the waiving party.
  </p>
  <br />
  <h2
    className="point-title"
  >
    27. Jurisdiction
  </h2>
  <p
    className="text"
  >
    The laws of the Provinces of Canada and the federal laws of Canada applicable therein govern these Terms and any dispute of any sort that might arise between you and us. Except as otherwise provided herein, you agree that any claim, whether at law or equity, arising out of or related to the provision of Services by us, regardless of the date or location of accrual of such dispute, will be within the exclusive jurisdiction of the courts of British Columbia, Canada.
  </p>
  <br />
</div>
`;
